src/CLImate.php

```diff
@@ -11,46 +11,46 @@
 use League\CLImate\Util\UtilFactory;
 
 /**
- * @method mixed black(string $str = null)
- * @method mixed red(string $str = null)
- * @method mixed green(string $str = null)
- * @method mixed yellow(string $str = null)
- * @method mixed blue(string $str = null)
- * @method mixed magenta(string $str = null)
- * @method mixed cyan(string $str = null)
- * @method mixed lightGray(string $str = null)
- * @method mixed darkGray(string $str = null)
- * @method mixed lightRed(string $str = null)
- * @method mixed lightGreen(string $str = null)
- * @method mixed lightYellow(string $str = null)
- * @method mixed lightBlue(string $str = null)
- * @method mixed lightMagenta(string $str = null)
- * @method mixed lightCyan(string $str = null)
- * @method mixed white(string $str = null)
+ * @method CLImate|mixed black(string $str = null)
+ * @method CLImate|mixed red(string $str = null)
+ * @method CLImate|mixed green(string $str = null)
+ * @method CLImate|mixed yellow(string $str = null)
+ * @method CLImate|mixed blue(string $str = null)
+ * @method CLImate|mixed magenta(string $str = null)
```

Comment:
I believe standard styling methods always return an instance of `CLImate`, I don't think the `mixed` is necessary is it?

Reply:
No, it is probably not. I just added 'CLImate|', mixed was there before. I think mixed can be removed.